lib/stormpath-sdk/resource/application.rb

```diff
@@ -47,6 +48,37 @@ def self.load composite_url
     end
   end
 
+  def create_id_site_url(options = {})
+    base = "https://" + DEFAULT_SERVER_HOST + "/sso"
```

Comment:
I would change this to use the data_store's base_url and replace the "/v1" with "/sso" to make it work fine on different environments like private deployments, shared enterprise and development environments.